seed/static/seed/partials/dataset_detail.html

```diff
@@ -41,7 +41,7 @@ <h1>{$:: dataset.name $}</h1>
                                 <td>{$:: f.created | date: 'MM/dd/yyyy hh:mm:ss a' $}</td>
                                 <td class="is_aligned_right">{$:: f.num_rows || 0 $}</td>
                                 <td>{$:: getCycleName(f.cycle) $}</td>
-                                <td><a id="data-mapping-{$:: $index $}" role="button" class="btn btn-sm btn-primary" ui-sref="mapping(::{importfile_id: f.id})" ng-disabled="::!f.num_rows">{$:: f.number_of_mappings $} {$:: 'Data Mapping' | translate $}</a></td>
+                                <td><a id="data-mapping-{$:: $index $}" role="button" class="btn btn-sm btn-primary" ui-sref="mapping(::{importfile_id: f.id})" ng-disabled="::!f.num_rows || !f.cached_second_to_fifth_row">{$:: f.number_of_mappings $} {$:: 'Data Mapping' | translate $}</a></td>
```

Comment:
`cache_second_to_fifth_row` is the property that is used in the following function, but will remain null for meter data ImportRecords